schematics/src/ng-add/schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "SchematicsAngularFire2",
+  "title": "Adds AngularFire to a project",
+  "type": "object",
+  "properties": {
+    "module": {
+      "type": "string",
+      "default": "app",
+      "description": "Allows specification of the declaring module.",
+      "alias": "m",
+      "subtype": "filepath"
+    },
+    "firebaseApp": {
+      "type": "string",
+      "description": "A custom FirebaseApp name.",
+      "alias": "f"
+    },
+    "all": {
+      "type": "boolean",
+      "description": "Add all AngularFire modules (auth, db, storage).",
+      "alias": "a",
+      "default": false
+    },
+    "path": {
+      "type": "string",
+      "format": "path",
+      "description": "The path to the project.",
```

Comment:
You should be more specific here. Do you mean the project's _root_, or something else?